src/main/main.c

```diff
@@ -155,11 +155,7 @@ static struct re global_re = {
 
 static void poll_close(struct re *re);
 
-#ifdef SOLARIS
-static pthread_once_t pt_once = {PTHREAD_ONCE_INIT};
-#else
 static pthread_once_t pt_once = PTHREAD_ONCE_INIT;
-#endif
 static pthread_key_t  pt_key;
```

Comment:
I think I added this ifdef SOLARIS thing a long time ago. In general this hunk is a good one.